pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -4,11 +4,12 @@
 , librsvg, cups
 
 , pname ? "kicad"
+, stable ? true
 , oceSupport ? false, opencascade
 , withOCCT ? true, opencascade-occt
 , ngspiceSupport ? true, libngspice
 , scriptingSupport ? true, swig, python3, python3Packages
-, debug ? false, valgrind
+, debug ? (!stable), valgrind
```

Comment:
We usually don't build debug symbols on hydra. One can do though in an override.

Reply:
do you mean hydra somehow interferes with the compilation / forces stripping?

Reply:
No. It is more a convention that we don't ship debug symbols in nixpkgs by default because of the size overhead.

Reply:
considering the debug symbols are 750M and our -unstable won't be nightly, i'm willing to default debug to false

does it still make sense to `separateDebugInfo` then?
i haven't managed to use the separate debug symbols from an nix-env install

Reply:
I don't think it provides any benefits in that case.
Gdb would usually pick up debug symbols from alternative (configured) path.
I think I used it at some point and the documentation was helpful for that: https://sourceware.org/gdb/current/onlinedocs/gdb/Separate-Debug-Files.html